src/std/env.ab

```diff
@@ -15,59 +16,71 @@ pub fun get_env_var(var: Text, file: Text = ".env"): Text {
     return ""
 }
 
+/// Load the env file in the environment
 pub fun load_env_file(file: Text = ".env"): Null {
     unsafe $export "\$(xargs < {file})" > /dev/null$
 }
 
+/// ?
 pub fun shell_isset(name: Text): Bool {
```

Comment:
return true if variable is set also return false

Reply:
Checks if a specified environment variable is set and is not empty.